### PR TITLE
Testing sbt-coursier 1.0.0-M1

### PR DESCRIPTION
**Pull request by alexarchambault**

### CODE REVIEW

**Comment by alexarchambault** [comment]
This is using the main branch of the plugin, which requires Java >= 7. Hope that'll be fine.

**Comment by alexarchambault** [comment]
Ok, I'll have to stage the java-6 branch of coursier too.

**Comment by fommil** [comment]
you can change the docker image to use `ensime/ensime:v2.x` to get JDK7 in the CI if that makes it easier for you. The windows CI uses Java 7 (which I think is all you care about anyway... in which case you can disable GNU/Linux builds by deleting `.drone.yml`)

**Comment by alexarchambault** [comment]
All checks were green! Relaunched to lower the odds of random failures.

**Comment by fommil** [comment]
sweet, I'll run the windows one a few times to see if it triggers anything.

File: project/EnsimeBuild.scala [hunk 1]
```diff
@@ -29,6 +29,12 @@ object EnsimeBuild extends Build {
        "io.spray" %% "spray-json" % "1.3.2"
     ),
 
+    // WORKAROUND https://github.com/ensime/ensime-emacs/issues/327
+    resolvers += Resolver.jcenterRepo,
```
Comment (fommil) [inline project/EnsimeBuild.scala]: these were moved

**Comment by fommil** [comment]
I guess the next thing to do is to publish a java 6 version, update here (reverting drone changes) and see what happens?

**Comment by alexarchambault** [comment]
About https://ci.appveyor.com/project/fommil/ensime-server-circ6/build/1.0.198 (and https://ci.appveyor.com/project/fommil/ensime-server-circ6/build/1.0.196): these look like random failure independant of coursier (don't they?).

**Comment by alexarchambault** [comment]
Like https://ci.appveyor.com/project/fommil/ensime-server-circ6/build/1.0.192 (master, so no coursier).

**Comment by fommil** [comment]
yeah, `DebugTest` is pretty flakey. I've got some magic command to run which will let us gather server logs for those failures, just need to find the time to test it out.

**Comment by fommil** [comment]
I've noticed that in M10 coursier is trying to use ascii colours when inside Emacs. I might be able to force sbt to use the dumb jline terminal.

**Comment by alexarchambault** [comment]
You mean in M11? It's using SBT logging now, yes.

**Comment by fommil** [comment]
oh cool, in M10 I think it was doing its own thing. I'll have to make sure sbt is set to dumb terminal. It's during the downloads, I think it's trying to show the progress bars.

**Comment by fommil** [comment]
this is amazing

**Comment by fommil** [comment]
you realise you've solved one of the biggest problems facing pretty much every scala developer? :grinning:

**Comment by alexarchambault** [comment]
Not on Central yet :-( I'll relaunch once it is.

**Comment by fommil** [comment]
looks like we're in business :smile: I'll close this and go back to my original PR with all the history in it.

**Pull request closed without merging**